<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>30</y>
      <width>191</width>
      <height>601</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Parameter</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>X-Null</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Y-Null</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>RMIN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>RMAX</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Center correction</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Guess for the Center</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="XNULLlineEdit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="YNULLlineEdit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="RMINlineEdit">
      <property name="geometry">
       <rect>
        <x>62</x>
        <y>120</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="RMAXlineEdit">
      <property name="geometry">
       <rect>
        <x>62</x>
        <y>150</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="DRlineEdit">
      <property name="geometry">
       <rect>
        <x>62</x>
        <y>180</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>DR</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="DRTlineEdit">
      <property name="geometry">
       <rect>
        <x>62</x>
        <y>270</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>RMAXT</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="RMAXTlineEdit">
      <property name="geometry">
       <rect>
        <x>62</x>
        <y>240</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="RMINTlineEdit">
      <property name="geometry">
       <rect>
        <x>62</x>
        <y>210</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>DRT</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>RMINT</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="YSCATlineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>400</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>YSCAT:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="XSCATlineEdit">
      <property name="geometry">
       <rect>
        <x>92</x>
        <y>370</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>XSCAT:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ANGLElineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>430</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>430</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Angle:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="UseCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>500</y>
        <width>87</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Usable</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>Set points first</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Whole area</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Top-Down</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Right-Left</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Selected area</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Integration Style</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Beam">
     <attribute name="title">
      <string>Beam</string>
     </attribute>
     <widget class="QLineEdit" name="DELTASlineEdit">
      <property name="geometry">
       <rect>
        <x>92</x>
        <y>110</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>CADIST</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="CADISTlineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>DELTAS</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Electron Beam Settings</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>80</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Wavelenght</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="WAVElineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="BeamPushButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Set for all </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="bla">
     <attribute name="title">
      <string>Adv.</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Y-Scale:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Resolution:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>X-Scale: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Settings for the Plate</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="PIXELlineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="XPIXFAlineEdit">
      <property name="geometry">
       <rect>
        <x>92</x>
        <y>70</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="YPIXFAlineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Misc</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>IRECOA:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="IRECOAlineEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>170</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>IRECOA2:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="IRECOA2lineEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>200</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>TUNEXP:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="TUNEXPlineEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>230</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>RADI:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="RADIlineEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>260</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>SEPLA:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="SEPLAlineEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>290</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>ISECT:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ISECTlineEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>320</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="AdvPushButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>360</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Set for all</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="picLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>121</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="StartRadioButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>650</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Point</string>
    </property>
    <attribute name="buttonGroup">
     <string>buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="EndRadioButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>650</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>End Point</string>
    </property>
    <attribute name="buttonGroup">
     <string>buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="centerRadioButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>650</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Center</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string>buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>650</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>x [pix], [mm]:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>670</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>y [pix], [mm]:</string>
    </property>
   </widget>
   <widget class="QLabel" name="XPIXlabel">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>650</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="YPIXlabel">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>670</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="XMMlabel">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>650</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="YMMlabel">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>670</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="AngleRadioButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>650</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Angle Point</string>
    </property>
    <attribute name="buttonGroup">
     <string>buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="IntegratePushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Integrate</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="IntegrateAllRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>All</string>
    </property>
    <attribute name="buttonGroup">
     <string>buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="IntegrteUsableRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>All usable</string>
    </property>
    <attribute name="buttonGroup">
     <string>buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="IntegrateSelectedRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Selected one</string>
    </property>
    <attribute name="buttonGroup">
     <string>buttonGroup_2</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1186</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenFile"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenFile">
   <property name="text">
    <string>Open File(s)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
